content/docs/2.0.0/iofogctl/application-yaml-spec.md

```diff
@@ -117,6 +119,17 @@ spec:
       bluetoothEnabled: true
       watchdogEnabled: false
       abstractedHardwareEnabled: false
+      upstreamRouters: [default-router]
+      networkRouter: ''
+      host: <Agent_host>
```

Comment:
somewhere we use 'horse-1' as a sample agent, somewhere we use <fill-your-agent-name>
we should unite it

Reply:
updated